mti_nma/steps/avgshape/avgshape.py

```diff
@@ -46,41 +42,55 @@ def run(self, **kwargs):
         shparam = Shparam()
         manifest_filepaths_rel2abs(shparam)
         df_sh = shparam.manifest.copy()
```

Comment:
For the future: This function should be converted to accepting a path to a csv or an already in memory dataframe I think?

Reply:
Sorry I'm not sure what you're saying here - I honestly used this formatting to mimic the inital demo repo (mti_mnist). Don't totally follow how you suggest this be different - did the manifest_filepaths_rel2abs function change to accepting different input types?

Reply:
Everything looks great to me. Besides this point that I don't fully understand. I will let you two figure this one out.

Reply:
This will be fixed in the next PR